docs/en/connector-v2/source/Doris.md

```diff
@@ -0,0 +1,54 @@
+# Doris
+
+> Doris source connector
+
+## Description
```

Comment:
Hi, we have recently formulated a new document format specification. You can get details from #2625.

Please modify your document according to the requirements of the new document format.

Thank you.